docs/server/project_setup.rst

```diff
@@ -0,0 +1,175 @@
+.. _project_setup:
+
+Create a Project
+================
+
+Now that you have the server up and running, this guide will help you get
+started with Pootle setup for getting translations.
+
+
+Assumptions
+-----------
+
+- We are setting up a project which consists of PO files and you have them on
+  hand.
+- You have a POT file with the strings you want to get translated.
+- The files are using the GNU layout, more about that later.
+
+
+.. _project_setup#add-new-project:
+
+Adding a new project
+--------------------
+
+.. _project_setup#place-translation-files:
+
+Place translation files
++++++++++++++++++++++++
+
+First task is to place the translation files for your new project in a location
+where Pootle can find them. Pootle writes and reads the translation files from
+the location defined by :setting:`POOTLE_TRANSLATION_DIRECTORY`. You can
+customize the location by altering this setting. By default it is the
+:file:`translations` directory within Pootle codebase.
+
+As told above, we will be using a GNU layout for the examples. A GNU layout
+means that a given project contains only translation files named by its
+language code. That means that within that project there are no directories,
+just files. This is the simplest layout possible. Below you can see an example
+with two projects using the GNU layout:
+
+.. code-block:: bash
+
+    `-- translations
+        `-- project1
+        |   |-- de.po
+        |   |-- fr.po
+        |   |-- gl.po
+        |   |-- pt_BR.po
+        |   `-- templates.pot
+        `-- project2
+            |-- af.po
+            |-- eu.po
+            |-- pt_BR.po
+            |-- templates.pot
+            `-- zu.po
+
+You might have noticed that among the regular translation files there are two
+files named :file:`templates.pot`. These are the master files that contain the
+original strings. Usually these master files contain English strings, but we
+find much less confusing to use ``templates`` instead of ``en`` (language code
+for English).
+
+Create a :file:`my-project` directory within
+:setting:`POOTLE_TRANSLATION_DIRECTORY` and place on it the translation files
+for your new project. Make sure you have a :file:`templates.pot` among those
+project translation files.
+
+
+.. _project_setup#create-new-project:
+
+Creating the project
+++++++++++++++++++++
+
+In the top right menu of the interface (or top left if you are using an RTL
```

Comment:
Avoid right and left and any mention of RTL.  Just say in the admin menu, we have a picture after all.